src/components/GenericModal.vue

```diff
@@ -223,6 +232,7 @@ export default {
         this.selectContent();
       }
       if (key !== 'Escape') return;
+      event.preventDefault();
```

Comment:
On Safari clicking the `Escape`key exits the full-screen mode, and I would like to avoid this when the modal is open, reason why I'm preventing the default behavior.